apps/new-dealer/translations/src/en/validation.json

```diff
@@ -151,9 +151,11 @@
     "equal": "Enter a nationality from the list that appears"
   },
   "first-authority-holders-nationality-second": {
+    "equal": "Enter a nationality from the list that appears",
     "same-first-nationality": "Second nationality cannot be the same as the first nationality"
```

Comment:
Is there a test to assert there is an error (don't assert on the error message, just that it's there) when the nationality is invalid?